frontend/src/components/ManageSchema.vue

```diff
@@ -57,21 +70,58 @@
         </v-tabs>
         <v-tabs-items v-model="tab">
           <v-tab-item transition="false" value="schema-attributes">
-            <v-card flat class="mt-2">
+            <v-card flat class="mt-2" v-show="!isEdit">
               <v-list-item
                 v-for="attribute in schema.schemaAttributeNames"
                 :key="attribute.id"
               >
                 <p class="grey--text text--darken-2 font-weight-medium">
                   {{ attribute }}
                   <v-icon
+                    :key="checkBoxGroup"
                     x-small
-                    v-if="attribute === schema.defaultAttributeName"
+                    v-show="isDefaultAttribute(attribute)"
                     >$vuetify.icons.asterisk</v-icon
                   >
                 </p>
               </v-list-item>
             </v-card>
+            <v-form v-show="isEdit">
+              <v-card flat class="mt-2">
+                <v-row>
+                  <v-col cols="8" class="py-0"> </v-col>
```

Comment:
Empty column? did you mean `<v-col \>`

Reply:
yes, due to spacing and header columns are in locales